LICENSE

```diff
@@ -1,21 +1,45 @@
-MIT License
+                        Bprotocol Foundation (Bancor) LICENSE
```

Comment:
MIT license should stay here

Reply:
I've reverted the LICENSE to its original version; is that good enough?

Reply:
yup